Fix EISGIT error

After ``npm install`` in `node_modules`:
![изображение](https://user-images.githubusercontent.com/142790/60260075-1dbddb00-98f2-11e9-874a-7f4aae8ec0ea.png)

if I don't remove `node_modules` and again do `npm install`, I get this:

```.sh
>npm i
npm ERR! path node_modules\karma-django-manage
npm ERR! code EISGIT
npm ERR! git node_modules\karma-django-manage: Appears to be a git repo or submodule.
npm ERR! git     node_modules\karma-django-manage
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
...
```

 .git file somehow made it into the release on NPM and is causing this issue.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,8 @@
     "name": "karma-django-manage",
     "main": "index",
     "description": "Call Django manage.py command before test runs.",
-    "version": "0.10.0",
+    "version": "0.10.1",
+    "files": ["index.js"],
     "repository": {
         "type": "git",
         "url": "http://github.com/100maketov/karma-django-manage"

